### DISCUSSION TITLE
Portfolio Generator

### DISCUSSION THREAD

**Discussion by vijaygupta18** [scaleracademy/open-source-projects#45] [Ideas]
We can make a site for auto generating Portfolios of users just by getting details from them by a form.
After submitting that form user will get his/her portfolio without worrying about html,css or javascript.
We can also store those details in database so that with a particular link user can access his auto build portfolio anytime.
For better understanding visit my demo project below.
Demo Link: https://portfoliogen.herokuapp.com/

**Comment by dewanshrawat15** [comment]
This seems great, however for long time usage, if the user needs to update from time to time, this seems like a lot of effort.

**Reply by vijaygupta18** [reply]
> This seems great, however for long time usage, if the user needs to update from time to time, this seems like a lot of effort.

Well we can generate some random numbers as a primary key and store in database.If in future user wants to update some  fields then user will have to enter that key and fill the form and then submit.It will get updated.

**Reply by KhadeejathulHumaira** [reply]
This sounds great . Will this website allows the user to select the colours of their choice or it will maintain a standard format 🤔

**Reply by vijaygupta18** [reply]
> This sounds great . Will this website allows the user to select the colours of their choice or it will maintain a standard format 🤔

I wish i could work on this project more. But yeah this feature you suggested can also be implemented.

**Reply by neel0976** [reply]
> > This seems great, however for long time usage, if the user needs to update from time to time, this seems like a lot of effort.
> 
> Well we can generate some random numbers as a primary key and store in database. If in future user wants to update some fields then user will have to enter that key and fill the form and then submit. It will get updated.

We could create a dynamic site which reads a directory which can have a pre determined format for everything like a project file which can have URl description and image site will read IT and display project cards accordingly.

**Comment by MrKrishnaAgarwal** [comment]
It's a great idea, can be implemented 👍🏻

**Comment by cy2000yadav** [comment]
It's awesome , definitely will be good thing to have!!

**Comment by sangita9853** [comment]
For sure a amazing idea!!

**Comment by kaivalya98** [comment]
Superb !!

**Comment by HackvistCharm** [comment]
It looks a very great idea, We can provide **framework supports** like we can provide Source code in React.js or Next.js, etc.
We can also create this project as a **command line tools** where it will asks interrogative questions and based on your answers, a portfolio website will be generated.

**Comment by Mann54** [comment]
It's great idea !
This is best help for non-technical person for make their portfolio !

**Comment by kuraakhilesh8230** [comment]
great idea!!

**Comment by Anishgupta0** [comment]
amazing idea !!

**Comment by Harshit1020** [comment]
For sure a amazing idea

**Comment by BhumikaJena** [comment]
Nice idea!

**Comment by kishanrajput23** [comment]
It's a great idea brother sometimes it becomes too hard to create a new portfolio from scratch

**Comment by PrakashGatiyala** [comment]
Nice one, Requirement for this can be Linkedin Profile, Coding platform Profile(Codechef, codeforces), Github repos/Projects. 
To automate, we need these access so that user don't have to manually fill details in form.

**Comment by imGaneshS** [comment]
Great Idea Bro!!

**Comment by sreejasaha19** [comment]
It's a great idea.

**Comment by Rimjhim20** [comment]
Nice idea.

**Comment by Rakesh9100** [comment]
Nice idea
